beetsplug/discogs.py

```diff
@@ -416,7 +416,9 @@ def get_album_info(self, result):
         albumtype = media = label = catalogno = labelid = None
         if result.data.get("formats"):
             albumtype = (
-                ", ".join(result.data["formats"][0].get("descriptions", []))
+                ", ".join(
+                    str(result.data["formats"][0].get("descriptions", []))
+                )
```

Comment:
Consider [a release](https://discogs.com/release/21390406-Various-Power-Workout-Volume-1) that has these `formats`

```python
[ins] In [19]: result.data["formats"]
Out[19]: 
[{'name': 'Vinyl',
  'qty': '1',
  'text': 'Pink Translucent Marbled',
  'descriptions': ['LP', 'Compilation', 'Limited Edition']}]
```

This would give you the following albumtype

```python
[ins] In [22]: ", ".join(str(result.data["formats"][0].get("descriptions", [])))
Out[22]: "[, ', L, P, ', ,,  , ', C, o, m, p, i, l, a, t, i, o, n, ', ,,  , ', L, i, m, i, t, e, d,  , E, d, i, t, i, o, n, ', ]"
```